packages/framework/lib/BotClient.ts

```diff
@@ -41,7 +41,7 @@ export class BotClient extends Client {
 
     async loadFile(result: any, dir: string, collection: Collection<string, any>): Promise<void> {
         const [filename, file] = result;
-        const name = filename.substring(0, filename.length - 2);
+        const name = filename.substring(0, filename.length - 3);
```

Comment:
I think instead of hardcoding the removal of the extension (cause then this fails in situations like .mjs and .cjs), we can do this
```javascript
const { parse } = require("path");
...
const name = parse(filename).name
```